js/demo_policy/test_data/sessions/oci-manifest-digest-lookup-success.yml

```diff
@@ -0,0 +1,7 @@
+- type: Exchange
+  request: |
+    !OciManifestDigest
+    image: ghcr.io/kubewarden/kubectl:v1.31.0
+  response:
+    type: Success
+    payload: '{"digest":"sha256:8bc1e63847a71fc93cd49e30a5639d04f1f1c797bbb1b3f20f2d343fbf95bf15"}'
```

Comment:
Sorry, I said something terribly stupid the other day when I told you to use crane to get the OCI digest and to use an image that is not part of docker.io because of the rate limiting. All the things I said do NOT matter, since we're using a recorded session... sorry, you can undo all the things I asked you to do